src/auction.vy

```diff
@@ -146,38 +156,16 @@ def _transfer_ownership(new_owner: address):
 
 
 event AuctionStarted:
-    """
-    @notice Emitted when a new auction starts.
-    @dev This event is triggered when an auction is initiated by the auctioneer.
-    @param lot The tokenId of the NFT being auctioned.
-    @param patron The address of the patron for whom the auctioneer is starting the auction. 
-    @param end_date The Unix timestamp when the auction will end.
-    """
     lot: indexed(uint256)
     patron: indexed(address)
     end_date: uint256
 
 event AuctionEnded:
-    """
-    @notice Emitted when an auction ends.
-    @dev This event is triggered either when the auction duration has elapsed or the auction is manually ended by the auctioneer.
-    @param lot The tokenId of the NFT auctioned.
```

Comment:
Why this is removing docs?

Reply:
Vyper doesnt allow natspec for events in the same way it allows for functions, or at all anymore.  And as far as I can see, the vyper team simply forgot or chose not to include it later on.

https://docs.vyperlang.org/en/stable/natspec.html

I guess nobody cared much about it :/